tox.ini

```
@@ -1,16 +1,17 @@
 [tox]
-envlist = {py37,py38,py39}-django{22,32},py310-django32
+envlist = {py38,py39,py310}-django{3.2,4.0,4.1,4.2},py310-django42
```

This can be simplified if all versions of python are compatible with all versions of Django:
```suggestion
envlist = {py38,py39,py310}-django{32,40,41,42}
```

(Additionally, I removed the "." so they correspond to the env names defined in `testenv`, below.)

---

👍🏽